docs/DeveloperGuide.md

```diff
@@ -121,9 +121,9 @@ How the parsing works:
 * All `XYZCommandParser` classes (e.g., `AddCommandParser`, `DeleteCommandParser`, ...) inherit from the `Parser` interface so that they can be treated similarly where possible e.g, during testing.
 
 ### Model component
-[<sub><sup>Back to top</sup></sub>](#table-of-contents)
```

Comment:
Any reason why was this removed?

Reply:
oh this is an error on my end! Shouldn't have been removed here! will re-add